contrib/vcloud/vcloudbenchmarkbase.py

```diff
@@ -85,6 +85,12 @@ def add_vcloud_args(self, vcloud_args):
             type=str,
             help="Specify files or paths that shall also be transferred and be made available to the run in the cloud.",
         )
+        vcloud_args.add_argument(
+            "--fresh-ivy-cache",
```

Comment:
The name and description of this option seem to be written from the point of view of the implementation, not of the user: The point is to not use the Ivy cache, the fact that Ivy still needs a temporary directory is an irrelevant detail for the user.

So the option should be named something like "no ivy cache" and the description should mention that this is useful for avoiding clashes due to concurrent access to the Ivy cache.